CMakeLists.txt

```diff
@@ -63,13 +63,19 @@ install(FILES
 install(DIRECTORY "include/" TYPE INCLUDE)
 
 if(EXPECTED_BUILD_TESTS)
-  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
-  set(CATCH_INSTALL_HELPERS OFF)
-  set(CATCH_BUILD_TESTING OFF)
-  set(CATCH_INSTALL_DOCS OFF)
-  FetchContent_Declare(Catch2 URL
-    https://github.com/catchorg/Catch2/archive/v2.13.10.zip) 
-  FetchContent_MakeAvailable(Catch2)
+  find_package(Catch2 2 QUIET)
+  if (Catch2_FOUND)
+    message(STATUS "Packaged version of Catch will be used.")
+  else()
```

Comment:
I'd drop `QUIET` the true-branch, then check `if (NOT Catch2_FOUND)` so that CMake itself reports the found version.

Reply:
`if (Catch2_FOUND)` can be dropped and replaced by `if (NOT Catch2_FOUND)` with downloader code.